src/test/java/com/amazon/opendistroforelasticsearch/sql/esintgtest/TypeInformationIT.java

```diff
@@ -0,0 +1,200 @@
+/*
+ *   Copyright 2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License").
+ *   You may not use this file except in compliance with the License.
+ *   A copy of the License is located at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   or in the "license" file accompanying this file. This file is distributed
+ *   on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ *   express or implied. See the License for the specific language governing
+ *   permissions and limitations under the License.
+ */
+
+package com.amazon.opendistroforelasticsearch.sql.esintgtest;
+
+import org.json.JSONArray;
+import org.json.JSONObject;
+import org.junit.Test;
+
+import static com.amazon.opendistroforelasticsearch.sql.util.MatcherUtils.rows;
+import static com.amazon.opendistroforelasticsearch.sql.util.MatcherUtils.schema;
+import static com.amazon.opendistroforelasticsearch.sql.util.MatcherUtils.verifyDataRows;
+import static com.amazon.opendistroforelasticsearch.sql.util.MatcherUtils.verifySchema;
+
+public class TypeInformationIT extends SQLIntegTestCase {
+
+    @Override
+    protected void init() throws Exception {
+        loadIndex(Index.ACCOUNT);
+        loadIndex(Index.ONLINE);
+    }
+
+    /*
+    numberOperators
+     */
+    @Test
+    public void testAbsWithIntFieldReturnsInt() {
+        JSONObject response = executeJdbcRequest("SELECT ABS(age) FROM " + TestsConstants.TEST_INDEX_ACCOUNT +
+                " ORDER BY age LIMIT 5");
+
+        verifySchema(response, schema("ABS(age)", null, "long"));
+        verifyDataRows(response,
+                rows(20),
+                rows(20),
+                rows(20),
+                rows(20),
+                rows(20));
```

Comment:
I think you're safe to remove all these `verifyDataRows()` and focus on what this IT want to test (the type in schema). The correctness of data rows should be covered by other IT.

Reply:
will do